### DISCUSSION TITLE
By default, the living do not hear the dead at sv_alltalk 0 This macro "fixes" this

### DISCUSSION THREAD

**Discussion by gameinside** [SmileYzn/PugMod-AMXX#41] [General]
in PugCore.sma 
In line 26
`#define alive_listen_dead_players`
From above  public plugin_init()

`#if defined alive_listen_dead_players
	#include <fakemeta>
#endif
#define GetTeam(%0) get_member(%0, m_iTeam)	`

below the line DisableHookChain(g_hRoundEnd = RegisterHookChain(RG_RoundEnd,"HOOK_RoundEnd",true)); Paste

`#if defined alive_listen_dead_players
	register_forward(FM_Voice_SetClientListening, "FwdSetClientListening", false);
#endif`

Higher or lower public client_disconnected(id,bool:bDropped,szReason[],iLen)  ; paste 

#if defined alive_listen_dead_players
public FwdSetClientListening(iReciever, iSender)
{
	if(iReciever == iSender)
		return FMRES_IGNORED;
	if(GetTeam(iReciever) != GetTeam(iSender))
		return FMRES_IGNORED;
	if(is_user_alive(iReciever) == is_user_alive(iReciever))
		return FMRES_IGNORED;
	
	engfunc(EngFunc_SetClientListening, iReciever, iSender, false);
	forward_return(FMV_CELL, false);
	return FMRES_SUPERCEDE;
}
#endif 

Now everything works, your teammates when the dead can talk to you or prompt

**Comment by SmileYzn** [comment]
Rehlds already have dead talk feature, if you do not want simple change at ESL and esl-ot config files.

[check it here](https://github.com/s1lentq/ReGameDLL_CS/wiki/sv_alltalk) 

Ps. I think these configs are wrong on pug 🤣

**Reply by gameinside** [reply]
Yes, I know) I just decided to share, and when is the new release?
